src/main/java/seedu/address/logic/commands/AddCustomerCommand.java

```diff
@@ -2,73 +2,64 @@
 
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_CUSTOMER;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_DRIVER;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
 
 import seedu.address.logic.commands.exceptions.CommandException;
 import seedu.address.model.Model;
 import seedu.address.model.person.Customer;
-import seedu.address.model.person.Driver;
 import seedu.address.model.person.Person;
 
 
 
 /**
  * Adds a person to the address book.
  */
-public class AddCustomerDriverCommand extends Command {
+public class AddCustomerCommand extends Command {
 
-    public static final String COMMAND_WORD = "add";
+    public static final String COMMAND_WORD = "addC";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a customer to the address book. "
             + "Parameters: "
-            + PREFIX_CUSTOMER + "CUSTOMER NAME | "
-            + PREFIX_DRIVER + " DRIVER NAME | "
+            + PREFIX_NAME + "CUSTOMER NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_ADDRESS + "ADDRESS "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
-            + PREFIX_CUSTOMER + "John Doe "
+            + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
             + PREFIX_TAG + "friends "
             + PREFIX_TAG + "owesMoney";
 
-    public static final String MESSAGE_SUCCESS = "New %1$s added: %2$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This %1$s already exists.";
+    public static final String MESSAGE_SUCCESS = "New Customer added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This Customer already exists.";
 
     private final Person toAdd;
-    private final String className;
     private final int toAddId;
 
     /**
      * Creates an AddCommand to add the specified {@code Person}
      */
-    public AddCustomerDriverCommand(Person person) {
+    public AddCustomerCommand(Person person) {
         requireNonNull(person);
         toAdd = person;
-        this.className = person.getClass().getSimpleName();
         this.toAddId = person.getId();
```

Comment:
Person can change to Customer already, since u return a Customer

Reply:
Changed.